scale/scale/settings.py

```diff
@@ -29,6 +29,10 @@
 # The full name for the Scale Docker image (without version tag)
 SCALE_DOCKER_IMAGE = 'geoint/scale'
 
+# The location of the config file containing Docker credentials
+# The URI value should be something like this 'http://marathon-ip:8080/v2/artifacts/.dockercfg'
```

Comment:
Marathon artifact store isn't available on a default DC/OS or Marathon deployment and is currently pretty limited in use. Suggest the following change:
`# The URI value should point to an externally hosted location such as a webserver or hosted S3 bucket. The value will be an http URL such as 'http://static.mysite.com/foo/.dockercfh`